test/sagemaker_tests/mxnet/inference/integration/local/test_hosting.py

```diff
@@ -27,6 +29,8 @@
 
 # The image should use the model_fn and transform_fn defined
 # in the user-provided script when serving.
+@pytest.mark.integration("SM_hosting")
```

Comment:
If the pytest markers that can be obtained through collect-reports are case sensitive, please use all lower-case or upper-case.

This will make it easier to search for all tests marked "sm_hosting", rather than searching for all upper-case and lower-case variations of "SM_hosting".

Reply:
SM_hosting is not the name of a marker, it is an argument to the "integration" marker and gets populated in a test report csv file. I will be factoring this into a constant anyway, however, and can keep it lower case.

Reply:
I changed to "hosting" -- i have left constants out of this PR due to large impact on SM tests